tool/tsh/mfa.go

```diff
@@ -669,7 +669,7 @@ func showOTPQRCode(k *otp.Key) (cleanup func(), retErr error) {
 	}
 	log.Debugf("Opened QR code via %q", imageViewer)
 	return func() {
-		if err := os.Remove(imageFile.Name()); err != nil {
+		if err := utils.RemoveSecure(imageFile.Name(), 3); err != nil {
```

Comment:
Mostly out of curiosity, do we have a source for "3" being a secure enough value here ?

Reply:
> Though they actually did a pass of zeroes, ones and then random

Is there a particular rationale for this, instead of _n_ random passes? I wonder if we should stick to the standard too.

Reply:
My rational was just keeping things simple. The DoD standard is from 1995 and is for wiping entire hard drives. In fact with many modern filesystems the file isn't even overwritten but rather a new file is created. So this is at best a mitigation. If folks feel strongly the standard can be implemented but it didn't feel like it added much value.

Reply:
No strong opinions on my part.